libdebug/utils/syscall_utils.py

```diff
@@ -55,9 +55,9 @@ def get_syscall_definitions(arch: str) -> dict:
 
 
 @functools.cache
-def resolve_syscall_number(name: str) -> int:
+def resolve_syscall_number(architecture: str, name: str) -> int:
```

Comment:
for example, here libcontext.arch is not used anymore. Why are we keeping libcontext.platform (=libcontext.arch)?

Reply:
Future proofing

Reply:
Rob...

Reply:
I have branches where `platform` is used, and I believe it to be an important distinction to make